editor/src/components/canvas/controls/select-mode/flex-gap-control.tsx

```diff
@@ -209,8 +209,8 @@ const GapControlSegment = React.memo<GapControlSegmentProps>((props) => {
         position: 'absolute',
         left: bounds.x,
         top: bounds.y,
-        width: bounds.width,
-        height: bounds.height,
+        width: bounds.width || 1,
```

Comment:
```suggestion
        width: bounds.width ?? 1,
```

Reply:
Yes please to this - please avoid using `||` for truthiness like this